PyClical build fails to find headers

Building PyClical can result in the following error, caused by the include paths `'.'` and `'..' being omitted`:
```
[   21s] g++ -std=c++11 -fno-strict-aliasing -Wno-unused -Wall -g -O3 -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -finline-limit=8000 --param large-function-insns=8000 --param max-inline-insns-auto=8000 --param max-inline-insns-single=8000 --param inline-unit-growth=200 --param large-function-growth=200 -funroll-loops -fvariable-expansion-in-unroller -DNDEBUG -Wno-unused-local-typedefs -Wno-deprecated-register -fno-check-new -DHAVE_BITS_WORDSIZE_H -D_GLUCAT_HAVE_BOOST_SERIALIZATION_ARRAY_WRAPPER_H -fexceptions -fPIC -I/usr/include/python3.10 -c PyClical.cpp -o build/temp.linux-x86_64-cpython-310/PyClical.o
[   21s] In file included from PyClical.cpp:1255:
[   21s] PyClical.h:34:10: fatal error: glucat/glucat_config.h: No such file or directory
[   21s]    34 | #include "glucat/glucat_config.h"
[   21s]       |          ^~~~~~~~~~~~~~~~~~~~~~~~
[   21s] compilation terminated.
[   21s] error: command '/usr/bin/g++' failed with exit code 1
[   21s] make[2]: *** [Makefile:534: PyClical.*.so] Error 1
```

--- a/pyclical/Makefile.am
+++ b/pyclical/Makefile.am
@@ -37,7 +37,7 @@ all: $(pyexec_DATA)
 $(pyexec_DATA): $(SETUP_SOURCE_LIST) $(SETUP).py setup_ext.py
 	ext_name="$(ext_name)" $(SETUP_SOURCE_PARAM) \
 	CXX="$(CXX)" CXXVERSION="$(CXXVERSION)" CFLAGS="" CXXFLAGS="$(EXTCXXFLAGS)" AM_CPPFLAGS="$(EXTAM_CPPFLAGS)" \
-	LDFLAGS="$(USER_LDFLAGS)" LIBRARIES="$(LIBS)" \
+	all_includes="$(all_includes)" LDFLAGS="$(USER_LDFLAGS)" LIBRARIES="$(LIBS)" \
 	$(PYTHON) $(SETUP).py build_ext --inplace
 
 if use_cython

--- a/pyclical/setup_ext.py
+++ b/pyclical/setup_ext.py
@@ -29,9 +29,11 @@ libraries = os.environ["LIBRARIES"].replace("-l", "")
 
 
 def setup_ext(ext_name, source):
+    all_includes_list = os.environ["all_includes"].replace("-I", "").split()
     ext = Extension(
-        ext_name,        # name of extension
-        sources=[source] # filename of our Cython source
+        ext_name,         # name of extension
+        sources=[source], # filename of our Cython source
+        include_dirs=all_includes_list
     )
     return ext
 

